static/app/components/createAlertButton.spec.jsx

```diff
@@ -41,21 +41,25 @@ describe('CreateAlertFromViewButton', () => {
     expect(onClickMock).toHaveBeenCalledTimes(1);
   });
 
-  it('disables the create alert button for members', () => {
+  it('disables the button for org-member', () => {
```

Comment:
org member has `alerts:write`? https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L2055

Reply:
Setting `sentry:alerts_member_write` to False will remove from org-members.

https://github.com/getsentry/sentry/blob/72989255ae95dd5d2ba789f920611020d6d7d3b7/src/sentry/models/organization.py#L640-L644